web/client-api/src/main/java/io/deephaven/web/client/api/JsTable.java

```diff
@@ -1762,8 +1770,8 @@ public void handleSnapshot(TableTicket handle, TableSnapshot snapshot) {
                 viewportRows.size());
     }
 
-
-    protected void processSnapshot() {
+    @JsIgnore
+    public void processSnapshot() {
```

Comment:
verify this is necessary, probably revert

Reply:
This is necessary, holdover from viewport debouncing. Post-188, we can probably remove this once and for all.